### PR TITLE
Init 27 create user role routes

### PR DESCRIPTION
**Pull request by IbrahimQumseya**

### CODE REVIEW

File: src/auth/users.respository.ts [hunk 1]
```diff
@@ -41,4 +43,35 @@ export class UsersRepository extends Repository<User> {
...
+      }
+    } catch (error) {
+      throw new ConflictException('check the enum input');
+    }
+  }
+
+  async getRolesForUser(): Promise<object> {
+    const roleForUsers = Object.keys(UserRole).map((key) => ({
+      Name: key,
+      Role: UserRole[key],
```
Comment (sergiutoma) [inline src/auth/users.respository.ts]: The convention applied so far was for all keys to start with a lowercase letter(camel case).
```
{
     name: key,
     role: UserRole[key],
}
```

**Review by sergiutoma — CHANGES_REQUESTED** [review]
The implementation looks good, if you can just change the `get roles route` to return roles in our convention format(camel case). 👏

File: src/auth/users.respository.ts (hunk 1 above)
Comment (IbrahimQumseya) [reply src/auth/users.respository.ts]: u mean to return like this ? 
```
[
    {
        "role": "user"
    },
    {
        "role": "admin"
    },
    {
        "role": "moderator"
    }
]
```

**Review by sergiutoma — APPROVED** [review]
Looks good! 👏

**Pull request merged**